enterprise/app/filter/filter_util.tsx

```diff
@@ -37,7 +37,7 @@ export type SortBy =
   | "cache-down"
   | "cache-up"
   | "cache-xfer";
-export type SortOrder = "asc" | "desc";
+export type SortOrder = "" | "asc" | "desc";
```

Comment:
I think it would be cleaner to leave `SortOrder` as-is and then change the call-site to `SortOrder | ""` or `SortOrder | null` to make it more clear that it's an optional sort order, rather than mixing "optional" and "present" values into the same type.

Reply:
Changed these and other filter parameters to be optional and set to undefined when not set, per offline discussion.